reference/commands/creator/new.rst

```diff
@@ -83,3 +85,22 @@ Creates a new package recipe template with a 'conanfile.py' and optionally,
   .. code-block:: bash
 
       $ conan new mypackage/1.0@myuser/stable -t -ciglg -ciglc -ciu https://api.bintray.com/conan/myuser/myrepo
+
+- Create files from a custom, predefined user template:
+
+  .. code-block:: bash
+
+      $ conan new mypackage/1.0 --file=templates/myconanfile.py
+
+
+  Conan will look for ``templates/myconanfile.py`` in the Conan local cache. Any folder is possible, but ``templates``
+  is recommended. These files can be installed and managed by :ref:`conan_config_install` command. The templates use the python strings format syntax:
+
+  .. code-block:: text
+
```

Comment:
This has been improved in https://github.com/conan-io/conan/pull/5267, and now includes Jinja syntax:

```python
class {{package_name}}Conan(ConanFile):
        name = "{{name}}"
        version = "{{version}}"
```